src/components/Checkout/OrderTotals.vue

```diff
@@ -1,27 +1,25 @@
 <template>
-	<div class="row">
```

Comment:
Just removed this row, was causing conflict with other elements on page that make it not uniform.